docs/nx-cloud/reference/nx-cloud-cli.md

```diff
@@ -1,5 +1,30 @@
 # `nx-cloud` CLI
 
+## npx nx-cloud login
+
+To provision a local personal access token to access Nx Cloud features run `npx nx-cloud login`. This will open your browser at the Nx Cloud application and after signing in will generate a personal access token and save it in a configuration file locally called `nxcloud.ini`. The format of this file will look like this:
```

Comment:
```suggestion
To provision a local personal access token to access Nx Cloud features run `npx nx-cloud login`. This will open your browser to the Nx Cloud application and after signing in will generate a personal access token and save it in a configuration file locally called `nxcloud.ini`. The format of this file will look like this:
```

Reply:
Where is that configuration file stored locally?  Is it placed in the root of the workspace?  Or your user directory?  How does the browser access your file system to save a configuration file?

Reply:
Order of precendence:

Windows:
We look to find this file within the %LOCALAPPDATA%/nxcloud directory

If it does not exist, we will create a new `nxcloud.ini` file there

Unix: 
We check to see if this file exists in the following order
1. `$XDG_CONFIG_HOME/nxcloud/nxcloud.ini`
2. `$HOME/config/nxcloud/nxcloud.ini`
3. `$HOME/.nxcloud.ini`

If we don't find an existing config file, we create one under `$HOME/config/nxcloud/nxcloud.ini`

Reply:
I'm just going to add that to the docs, @lourw